api/pkg/transformer/feast/transformer.go

```diff
@@ -181,31 +208,113 @@ func (t *Transformer) getFeastFeature(ctx context.Context, tableName string, req
 		features = append(features, feature.Name)
 	}
 
-	feastRequest := feast.OnlineFeaturesRequest{
-		Project:  config.Project,
-		Entities: entities,
-		Features: features,
-	}
-	t.logger.Debug("feast_request", zap.Any("feast_request", feastRequest))
-
-	startTime := time.Now()
-	feastResponse, err := t.feastClient.GetOnlineFeatures(ctx, &feastRequest)
-	durationMs := time.Now().Sub(startTime).Milliseconds()
+	feastFeature, err := t.feastFeatureBatchFetching(ctx, entities, config, features)
```

Comment:
Maybe `getFeatureFromFeast` is better name here. `feastFeatureBatchFetching` imply that there is batching, however caller shouldn't know about the implementation details i.e. batching